src/app/saga.ts

```diff
@@ -280,6 +283,10 @@ export function* handleDeepLink(action: OpenDeepLink) {
     const pathParts = rawParams.path.split('/')
     if (rawParams.path.startsWith('/pay')) {
       yield call(handlePaymentDeeplink, deepLink)
+    } else if (rawParams.path.startsWith('/payment')) {
```

Comment:
maybe this condition should go ahead of the previous one that checks for `/pay`? seems like a path starting with `/payment` also starts with `/pay` so does this block get triggered?

Reply:
Order changed in 93d3cbb9bd74fa53dbf9a1d6db2de055ddd939bf!